File: src/main/java/resonantinduction/core/fluid/FluidColored.java
package resonantinduction.core.fluid;

import net.minecraftforge.fluids.Fluid;

public class FluidColored extends Fluid
{
	private int color = 0xFFFFFF;

	public FluidColored(String fluidName)
	{
		super(fluidName);
	}

	public int getColor()
	{
		return color;
	}

	public void setColor(int newColor)
	{
		color = newColor;
	}
}

File: src/main/scala/resonantinduction/core/resource/content/FluidColored.scala
package resonantinduction.core.resource.content

import net.minecraftforge.fluids.Fluid

class FluidColored(fluidName: String) extends Fluid(fluidName)
{
  var color = 0xFFFFFF

  def setColor(color: Int): FluidColored =
  {
    this.color = color
    return this
  }

  override def getColor = color
}